shared/getNumberOfPeople.ts

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
extension/utils/getNumberOfPeople.ts - dodac typowanie
website/utils/getNumberOfPeople.ts

Reply:
Resolved in 5482e6b0ad979ce6f0f89bb295128bf7365d09fa